pkgs/development/interpreters/joker/default.nix

```diff
@@ -11,7 +11,7 @@ buildGoModule rec {
     sha256 = "03lbvxmkn0rr7yv7kj6c4dh4zayiwbhrkqbz2m22dw568rjbp8az";
   };
 
-  modSha256 = "1mylgim9nm0v5wybgzi74nqzmvzwzws0027wzl86dbj5id83ab9v";
```

Comment:
I wonder if for a smoother upgrade process we should rename this to a new attribute and remove the old one after the next release.

Reply:
We had a similar change in Rust and we opted to add a build-time check in the main derivation to check the source format.

Reply:
Yes. However having let's say a `vendorSha256` would be cleaner since it's allow us to implement the `vendorSha256 = null;` case, as `modSha256 = null;` is already in use. Also it allows to set both for migration from the old sum to the new sum.

Reply:
Flipped to vendorSha256.